### ISSUE TITLE
Header/footer special symbols tooltip closes too quickly

### ISSUE THREAD

**Issue by Riley-Sullivan** [musescore/MuseScore#23241]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Bug description

The header/footer tooltip disappears after about 5 seconds, too quickly to be able to view all the options. Other tooltips in the app are visible for a full 10 seconds.

### Steps to reproduce

1. Go to Format > Style > Header & Footer.
2. Hover the mouse cursor over any of the header or footer fields to display the special symbols tooltip, and keep the cursor there.
3. See issue: the tooltip disappears after about 5 seconds.

Note that if you hover over the "Show on first page" checkbox, its tooltip remains for about 10 seconds.

### MuseScore Version

4.3.1

### Regression

No.

### Operating system

macOS 12.7.2

### Additional context

Same behavior in MS3.
It's actually a MS2 regression. In MS2, the tooltip stated open for over a minute.

**Comment by Jojo-Schmitz** [comment]
Culprit code:
https://github.com/musescore/MuseScore/blob/a0639b1a5d4904c7c117cffff7af71ada24c50b6/src/notation/view/widgets/editstyle.cpp#L1404-L1407

**Comment by Jojo-Schmitz** [comment]
If changing that to 10 seconds is desired, you can assign the issue to me

**Comment by Jojo-Schmitz** [comment]
That toolTop showing when hovering over "Show on first page" seems a bug to me (and doesn't happen in Mu3)
Same for all the other checkboxes (for some it happens in Mu3 too).

**Comment by Eism** [comment]
I would suggest not limiting the display time of the tooltip for these fields at all, because the text there is very long and it seems to me that 10 seconds will not be enough for the user to read

Two minutes looks good to me.

**Comment by Jojo-Schmitz** [comment]
I disagree: that huge toolTip covers large parts of the screen and also those where you try to input stuff.

**Comment by Jojo-Schmitz** [comment]
Wopuld be different if it were a 'normal' dialog window you could grab and move aside. But it isn't ...

**Comment by cbjeukendrup** [comment]
IMO, the real solution is not making it a tooltip at all, but simply an always-visible text area at the bottom of the dialog. We could make it expandable/collapsible, but I don't even think that's necessary, because it won't get in the way in any way.
(Of course, this would require that we make that page scrollable, but I think it already is.)

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/368199

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/368418

**Comment by cbjeukendrup** [comment]
How about something like this?
<img width="906" alt="Scherm­afbeelding 2024-09-09 om 18 26 10" src="https://github.com/user-attachments/assets/02cf5172-c7f7-4037-9be0-dd6d61a16fc3">

**Comment by Jojo-Schmitz** [comment]
Looks good to me

**Comment by Jojo-Schmitz** [comment]
I believe to have seen a commit, some code change, for this, some hour ago but can't find it now?

**Comment by Riley-Sullivan** [comment]
I like that Casper. Does your fix also remove the tooltip completely? It probably should, right?

**Comment by cbjeukendrup** [comment]
Yes, it does.
